dll/win32/kernel32/kernel32_vista/sync.c

```diff
@@ -3,52 +3,6 @@
 #define NDEBUG
 #include <debug.h>
 
-VOID
-NTAPI
-RtlInitializeConditionVariable(OUT PRTL_CONDITION_VARIABLE ConditionVariable);
```

Comment:
This part (and the other one in the NDK) should better be done separately by committing first PR #6806 and then rebasing the current PR on top of master.

Reply:
Done, #6806 is ready